FindDistance/src/ua/goit/alg/FindDistance.java

```diff
@@ -0,0 +1,35 @@
+package ua.goit.alg;
+
+public class FindDistance {
+    public static void main (String[] args) {
```

Comment:
This overall looks cool but please drop main method - you've got unit tests for the demonstration